web/src/repogroups/RepogroupPage.scss

```
@@ -10,6 +10,7 @@
     &__logo {
         width: 3rem;
         height: 3rem;
+        margin: 8px;
```

Can you use rem instead of pixels? 
```suggestion
        margin: 0.5rem;
```

---

Alternatively, you can use a bootstrap margin class `mr-2`.
https://about.sourcegraph.com/handbook/engineering/web/styling#spacing